### ISSUE TITLE
Clean the branches for robot_dart and limbo

### ISSUE THREAD

**Issue by jbmouret** [resibots/blackdrops#8]
We should not depend on some unmerged branch of Limbo and robot_dart. Either we merge them, or we remove the dependency.

**Comment by costashatz** [comment]
Will do next week! This mean a new release for blackdrops too (breaks backward compatibility)..

**Comment by costashatz** [comment]
@jbmouret and @BoltzmannBrain can you please check that the **dev** branch compiles and works in MacOS? Be careful to use the **multi_robot** branch of robot_dart when doing the advanced installation (I do not want to merge before I am sure that it works).. Thanks a lot!

**Comment by BoltzmannBrain** [comment]
@costashatz will do, but alright if I do so later today? I'm traveling Boston-London-SF these few days.

**Comment by costashatz** [comment]
Sure..no problem!

**Comment by BoltzmannBrain** [comment]
The blackdrops build on the dev branch failed to find DART. I then went back to the robot_dart step of the advanced install to use the multi_robot branch, but this fails:
```
$ git status
HEAD detached at origin/multi_robot
nothing to commit, working tree clean
$ ./waf configure
Setting top to                           : /Users/bb/robot_dart
Setting out to                           : /Users/bb/robot_dart/build
Checking for 'clang++' (C++ compiler)    : /usr/bin/clang++
Checking for 'clang' (C compiler)        : /usr/bin/clang
Checking boost includes                  : 1_67
Checking boost libs                      : ok
Checking for Eigen                       : /usr/local/include/eigen3
Checking for DART includes (including io/urdf) : 6.2.1 in /usr/local/include
Checking for DART gui includes                 : /usr/local/include
Checking for DART libs (including io/urdf)     : ['dart', 'dart-utils', 'dart-utils-urdf']
DART: Checking for Assimp                      : /usr/local/include
DART: Checking for Bullet Collision libs       : libs: ['LinearMath', 'BulletCollision', 'dart-collision-bullet'], bullet: /usr/local/include/bullet
DART: Checking for gui libs                    : ['dart-gui', 'dart-gui-osg']
DART: Checking for OSG (optional)              : ['osg', 'osgViewer', 'osgManipulator', 'osgGA', 'osgDB', 'osgShadow', 'OpenThreads']
Checking for hexapod_controller includes       : Not found
['-Wall', '-std=c++11', '-O3', '-march=native', '-g', '-faligned-new']
'configure' finished successfully (0.124s)
$ sudo ./waf install
Password:
Waf: Entering directory `/Users/bb/robot_dart/build'
Could not find 'build/libRobotDARTSimu.a' in /Users/bb/robot_dart
```

**Comment by costashatz** [comment]
> The blackdrops build on the dev branch failed to find DART. I then went back to the robot_dart step of the advanced install to use the multi_robot branch, but this fails:

Makes sense because I had a minor bug in the advanced installation. I pushed a fix that should do it.. Let me know and thanks for trying..!

**Comment by BoltzmannBrain** [comment]
Okay tried again... Getting `fatal error: 'limbo/model/multi_gp.hpp' file not found`, which I notice is because the `spt` branch of limbo does not have this file -- that's the specified branch in the blackdrops advanced install guide. Checking out limbo master, I'm now getting `fatal error: 'simple_nn/neural_net.hpp' file not found` for the places you're trying to use that header file: `#include <simple_nn/neural_net.hpp>`.

**Comment by costashatz** [comment]
> Okay tried again...

Thanks for trying..

> that's the specified branch in the blackdrops advanced install guide

Yeap. That was wrong. We need the `master` branch.. Pushed a fix..

> I'm now getting `fatal error: 'simple_nn/neural_net.hpp' file not found` for the places you're trying to use that header file: `#include <simple_nn/neural_net.hpp>`.

Have you installed [simple_nn](https://github.com/resibots/simple_nn)? This is written in the [docs](https://github.com/resibots/blackdrops/blob/dev/docs/advanced_installation.md#installing-simple_nn) (you would need to re-configure)...

**Comment by costashatz** [comment]
This should be fixed by #11 .. If something remains, feel free to open a new issue..
